Lib/fontTools/pens/ttGlyphPen.py

```diff
@@ -82,6 +82,30 @@ def addComponent(self, glyphName, transformation):
     def glyph(self, componentFlags=0x4):
         assert self._isClosed(), "Didn't close last contour."
 
+        almost2 = 1.99993896484375  # F2Dot14 0b1.11111111111111
+        eps = .5 / (1 << 14)
+
+        def closeTo2(value):
```

Comment:
why do we need to define `closeTo2` every time one calls the `.glyph()` method?
you could move it to the global scope and call it `_closeTo2` along with the two other constant values.